calcom/cal.com
# fix: Calendar View

## What does this PR do?



Fixes https://github.com/calcom/cal.com/issues/10428

TODO

- [x] Fix type errors



## How to test this?

1) Create availability in IST Timezone ( 9 -5 , mon-fri except wed)
2) Set your system's time to New York
3) go to booking page and select weekly view to verify if dates and slots are correct

## Type of change
